src/degeneracy.jl

```diff
@@ -154,5 +154,5 @@ function k_corona(g::AbstractGraph, k; corenum=core_number(g))
     kcoreg = g[kcore]
     kcoredeg = degree(kcoreg)
 
-    return kcore[findin(kcoredeg, k)]
+    return kcore[findall(occursin(k), kcoredeg)]
```

Comment:
If `k` is an `Int` here, then `occursin(k)` is just `x->x==k`

Reply:
There is an `equalto` which does exactly what we want

https://github.com/JuliaLang/julia/pull/24673/files#diff-a21a7fc275830bf9efb5b7c17c4fb98eR834